src/util/Utils.cpp

```diff
@@ -551,9 +458,13 @@ void Utils::updateSLAInformation(PerformanceSQLiteDBInterface perfSqlite, std::s
     }
 }
 
-void Utils::copyToDirectory(std::string currentPath, std::string copyPath) {
+int Utils::copyToDirectory(std::string currentPath, std::string copyPath) {
     std::string command = "mkdir -p " + copyPath + "&& " + "cp " + currentPath + " " + copyPath;
-    system(command.c_str());
+    int status = system(command.c_str());
+    if (status != 0) {
+        util_logger.error("copyToDirectory failed with code " + std::to_string(status));
```

Comment:
What do we mean by `copyToDirectory` ? Can we have more descriptive error message here?

Reply:
It's in the function `copyToDirectory` and it has failed. What additional information can we include there in addition to the exit code?

Reply:
Instead of `copyToDirectory` shall we use `Copying data to directory` ?

Reply:
Added current path and new path to error message

Reply:
Ok